### DISCUSSION TITLE
Problem with Galpy and Spyder

### DISCUSSION THREAD

**Discussion by edan0314** [jobovy/galpy#699] [Q&A]
Dear Jo Bovy and community,
Recently, I had to uninstall my Anaconda and when I had to reinstall it, along with Galpy, my Spyder app started to show an error message. Since I needed to use it for other purposes, I did a second reinstallation and didn't install Galpy this time, but I need to make use of it, but I would rather not deal with errors anymore. Somebody knows what could be the cause?

**Comment by jobovy** [comment]
I don't use Spyder, but maybe it would help if you pasted the error message that you're getting here?
